System/RTETools.cpp

```diff
@@ -198,4 +201,35 @@ namespace RTE {
 			return RoundFloatToPrecision((roundingBuffer / precisionMagnitude), precision);
 		}
 	}
+
+	unsigned long TrueAlphaBlender(unsigned long x, unsigned long y, unsigned long n) {
```

Comment:
Not sure this belongs in RTETools, it's a very specific chunk of code. I'm not too sure where else it should be, but RTETools should have generally widely useful functionality, not API-interaction-hacks